doc/api/quic.md

```diff
@@ -0,0 +1,2276 @@
+    recently validated addresses are remembered. Setting `validateAddressLRU`
+    to `true`, will enable the `validateAddress` option as well. Default:
+    `false`.
+
+The `net.createQuicSocket()` function is used to create new `QuicSocket`
+instances associated with a local UDP address.
+
+### Class: QuicEndpoint
+<!-- YAML
+added: REPLACEME
+-->
+
+The `QuicEndpoint` wraps a local UDP binding used by a `QuicSocket` to send
+and receive data. A single `QuicSocket` may be bound to multiple
+`QuicEndpoint` instances at any given time.
+
+Users will not create instances of `QuicEndpoint` directly.
+
+#### quicendpoint.addMembership(address, iface)
+<!-- YAML
+added: REPLACEME
+-->
+
+* `address` {string}
+* `iface` {string}
+
+Tells the kernel to join a multicast group at the given `multicastAddress` and
+`multicastInterface` using the `IP_ADD_MEMBERSHIP` socket option. If the
+`multicastInterface` argument is not specified, the operating system will
+choose one interface and will add membership to it. To add membership to every
+available interface, call `addMembership()` multiple times, once per
+interface.
+
+#### quicendpoint.address
+<!-- YAML
+added: REPLACEME
+-->
+
+* Type: Address
+
+An object containing the address information for a bound `QuicEndpoint`.
+
+The object will contain the properties:
+
+* `address` {string} The local IPv4 or IPv6 address to which the `QuicEndpoint` is
+  bound.
+* `family` {string} Either `'IPv4'` or `'IPv6'`.
+* `port` {number} The local IP port to which the `QuicEndpoint` is bound.
+
+If the `QuicEndpoint` is not bound, `quicendpoint.address` is an empty object.
+
+#### quicendpoint.bound
+<!-- YAML
+added: REPLACEME
+-->
+
+* Type: {boolean}
+
+Set to `true` if the `QuicEndpoint` is bound to the local UDP port.
+
+#### quicendpoint.closing
+<!-- YAML
+added: REPLACEME
+-->
+
+* Type: {boolean}
+
+Set to `true` if the `QuicEndpoint` is in the process of closing.
+
+#### quicendpoint.destroy(\[error\])
+<!-- YAML
+added: REPLACEME
+-->
+
+* `error` {Object} An `Error` object.
+
+Closes and destroys the `QuicEndpoint` instance making it usuable.
+
+#### quicendpoint.destroyed
+<!-- YAML
+added: REPLACEME
+-->
+
+* Type: {boolean}
+
+Set to `true` if the `QuicEndpoint` has been destroyed.
+
+#### quicendpoint.dropMembership(address, iface)
+<!-- YAML
+added: REPLACEME
+-->
+
+* `address` {string}
+* `iface` {string}
+
+Instructs the kernel to leave a multicast group at `multicastAddress` using the
+`IP_DROP_MEMBERSHIP` socket option. This method is automatically called by the
+kernel when the socket is closed or the process terminates, so most apps will
+never have reason to call this.
+
+If `multicastInterface` is not specified, the operating system will attempt to
+drop membership on all valid interfaces.
+
+#### quicendpoint.fd
+<!-- YAML
+added: REPLACEME
+-->
+
+* Type: {integer}
+
+The system file descriptor the `QuicEndpoint` is bound to. This property
```

Comment:
Informational at this point. Likely to be removed because it cannot be used directly. In fact, it's dangerous to do so. What does need to be understood, however, is that every `QuicEndpoint` consumes a file descriptor.

Reply:
+1 to remove